compose.yaml

```diff
@@ -0,0 +1,129 @@
+networks:
+  e2e-pod:
+    driver: bridge
+
+services:
+  searchdb-ocp:
```

Comment:
In the vein of "one template to rule them all", ideally we should be setting up both Opensearch instances. (We need frontend ILAB tests and backend OCP tests.)

It also occurs to me that we've got two `snapshot.tar.gz` files in different directories, which is rather unclean. They should be in the same place, with distinct names like `ocp_snapshot.tar.gz` and `ilab_snapshot.tar.gz`. (Sigh.)

Since I'm working now on rebuilding an ILAB snapshot, maybe I'll move it and rename it. Although I think we should have them under `tests/functional`, either in `setup` where I put mine or in a new `tests/functional/fixtures`. 🤔

Reply:
Yeah, I renamed yours but left it in the same place, and moved mine there in #225; I'd like to make this more consistent, but it can wait.